### ISSUE TITLE
Some Milestones display #task_number instead of name in Calendar

### ISSUE THREAD

**Issue by quad10** [bardsoftware/ganttproject#1597]
When preference is set display Milestones in the calendar band, 
most milestones are displayed with their Task name, 
but **some milestones are displayed with their task number** (#232 as example).
This seems to be caused by the presence of **ACCENTED CHARACTERS** in the task name like é ê è...

The task name displays properly in the task list though. So it should be "easy" to fix the issue in the calendar band.
Thanks for the great work.

**Comment by dbarashev** [comment]
Can you share a sample project?

**Comment by quad10** [comment]
Here is a sample file.
While preparing it, I found that my diagnostic is wrong.
Some milestones with accented characters show with their name.

[GanttProject milestones.zip](https://github.com/bardsoftware/ganttproject/files/2371084/GanttProject.milestones.zip)

**Comment by quad10** [comment]
I agree, task name, even truncated, is probably better since GanttProject allows users to work without having to know the task id.
